regulations/generator/sidebar/analyses.py

```diff
@@ -42,10 +42,11 @@ def fetch_data(self, http_client):
         """Retrieves SxSEntry data from the server"""
         if 'Interp' in self.label_id:
             reg_part = self.label_parts[0]
-            data = http_client.layer(
-                'analyses', reg_part + '-Interp', self.version)
+            doc_id = '{}/{}-Interp'.format(self.version, reg_part)
+            data = http_client.layer('analyses', 'cfr', doc_id)
```

Comment:
Minor, but lines 46 and 49 are now identical. Can we move them out of the conditional?